src/chrome/content/rules/Hatena.xml

```diff
@@ -55,7 +55,7 @@
 
 	<target host="*.hatena.ne.jp" />
 	<target host="*.st-hatena.com" />
-
+	<target host="cdn.api.b.hatena.ne.jp" />
```

Comment:
Thanks for the update! It looks like I was a bit too fast in my previous comment: This target host is not actually necessary, since it is covered by the wildcard target in line 56; would you mind removing it again? Only the rule that you added is necessary.